src/Microsoft.Extensions.Logging.Testing/Microsoft.Extensions.Logging.Testing.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
```

Comment:
Yeah I'm ok with this because it doesn't really change the situation. It's just very odd that we would do this - I doubt we did it knowingly to begin with.